对单个下载任务进行频繁开始，取消下载操作会报空指针异常！

### OkDownload Version

v1.0.3

### Problem Describe

I occur some problem, I also search on [issues][issues-url] and [wiki][wiki-url], I even checked the source code, but it can't help, so my real problem is:...

### Log

There is the whole log from LogCat when I occur the problem(I know we can use [this tool][okcat-wiki-url] to let log more readable):

```
```

[issues-url]: https://github.com/lingochamp/okdownload/issues
[wiki-url]: https://github.com/lingochamp/okdownload/wiki
[okcat-wiki-url]: https://github.com/lingochamp/okdownload/wiki/Debug-OkDownload

`06-05 11:33:29.448 363-363/com.questfree.duojiao E/haha: reStart: 開始任務
06-05 11:33:29.449 363-363/com.questfree.duojiao E/Download: DownloadStartEvent开始下载
06-05 11:33:29.449 15488-15652/? I/System.out: [getprop, ro.product.cpu.abi]
06-05 11:33:29.449 15488-15652/? I/System.out: null
06-05 11:33:29.449 15488-15652/? I/System.out: null
06-05 11:33:29.450 15488-15652/? I/System.out: Calling by::className:com.tencent.tbs.common.utils.DeviceUtils  MethodName:getDeviceCpuabi
06-05 11:33:29.453 363-363/com.questfree.duojiao E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.473 363-363/com.questfree.duojiao E/haha: 更新状态: 1
06-05 11:33:29.487 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 1
06-05 11:33:29.487 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.499 363-12503/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.557 363-11194/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.557 4212-4212/? I/TrafficMonitor: update:rxPkts:2494,txPkts:2363,rxBytes:2961714,txBytes:153181
06-05 11:33:29.557 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:29.558 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:29.567 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.588 363-15662/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.588 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.590 363-13327/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.602 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.612 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:29.613 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.627 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.658 363-11194/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.668 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:29.669 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.681 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:29.681 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.689 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:29.694 363-363/com.questfree.duojiao E/haha: STATUS_CONNECTED
06-05 11:33:29.696 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:29.697 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.709 363-363/com.questfree.duojiao E/QueueController: taskEnd: CANCELED
06-05 11:33:29.709 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:29.709 363-363/com.questfree.duojiao E/haha: 更新状态: 4
06-05 11:33:29.747 363-363/com.questfree.duojiao E/haha: onEvent: 4
06-05 11:33:29.749 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 4
06-05 11:33:29.785 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.889 363-363/com.questfree.duojiao E/haha: reStart: 開始任務
06-05 11:33:29.893 363-363/com.questfree.duojiao E/Download: DownloadStartEvent开始下载
06-05 11:33:29.896 363-363/com.questfree.duojiao E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.897 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.911 363-363/com.questfree.duojiao E/haha: 更新状态: 1
06-05 11:33:29.925 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 1
06-05 11:33:29.939 363-12503/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.972 363-15662/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.005 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.007 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.008 363-11194/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.010 363-13327/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.019 363-15662/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.025 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.047 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.049 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.056 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.069 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.070 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.080 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.080 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.092 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.096 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.108 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.117 363-363/com.questfree.duojiao E/haha: STATUS_CONNECTED
06-05 11:33:30.118 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:30.139 363-363/com.questfree.duojiao E/QueueController: taskEnd: CANCELED
06-05 11:33:30.139 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:30.139 363-363/com.questfree.duojiao E/haha: 更新状态: 4
06-05 11:33:30.161 363-363/com.questfree.duojiao E/haha: onEvent: 4
06-05 11:33:30.163 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 4
06-05 11:33:30.191 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.306 363-363/com.questfree.duojiao E/haha: reStart: 開始任務
06-05 11:33:30.307 363-363/com.questfree.duojiao E/Download: DownloadStartEvent开始下载
06-05 11:33:30.310 363-363/com.questfree.duojiao E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.320 363-363/com.questfree.duojiao E/haha: 更新状态: 1
06-05 11:33:30.332 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 1
06-05 11:33:30.332 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.352 363-12503/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.388 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.416 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.431 363-13531/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.434 363-15662/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.434 363-13327/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.439 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.455 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.456 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.456 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.472 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.501 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.501 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.515 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.516 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.528 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.531 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.541 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.548 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:30.550 4602-4602/? I/NetworkSpeedManagerEx:  value = 1601218.6
06-05 11:33:30.551 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.5M/s
06-05 11:33:30.551 4602-4602/? I/NetworkSpeedView: /update(), speed=1.5M/s
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: expired arrive. level:-1
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: during_ms:2002,period_ms:2000
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: count:980,rx_sum:2494,tx_sum:2363,rxBytes:2961714,during_ms:2002,rx_speed:1479377.6223776226,tx_speed:76513.98601398602,rto:1.0554380025391452
06-05 11:33:30.562 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:30.563 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.641 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.658 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.708 363-363/com.questfree.duojiao E/haha: STATUS_CONNECTED
06-05 11:33:30.709 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:30.721 363-363/com.questfree.duojiao E/QueueController: taskEnd: CANCELED
06-05 11:33:30.722 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:30.722 363-363/com.questfree.duojiao E/haha: 更新状态: 4
06-05 11:33:30.760 363-363/com.questfree.duojiao E/haha: onEvent: 4
06-05 11:33:30.762 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 4
06-05 11:33:30.852 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.903 363-363/com.questfree.duojiao E/haha: reStart: 開始任務
06-05 11:33:30.904 363-363/com.questfree.duojiao E/Download: DownloadStartEvent开始下载
06-05 11:33:30.907 363-363/com.questfree.duojiao E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.911 363-363/com.questfree.duojiao E/haha: 更新状态: 1
06-05 11:33:30.930 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 1
06-05 11:33:30.946 363-373/com.questfree.duojiao I/art: Background sticky concurrent mark sweep GC freed 293629(15MB) AllocSpace objects, 25(500KB) LOS objects, 19% free, 62MB/78MB, paused 8.949ms total 133.886ms
06-05 11:33:30.953 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.960 363-12503/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.003 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.030 363-11194/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.031 363-13327/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.035 363-13531/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.046 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.048 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.065 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:31.066 363-363/com.questfree.duojiao E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.077 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.092 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:31.096 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.114 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:31.116 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:31.122 363-363/com.questfree.duojiao E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.141 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 5
06-05 11:33:31.145 363-363/com.questfree.duojiao E/haha: STATUS_CONNECTED
06-05 11:33:31.146 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:31.167 363-363/com.questfree.duojiao E/QueueController: taskEnd: CANCELED
06-05 11:33:31.168 363-363/com.questfree.duojiao E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:31.168 363-363/com.questfree.duojiao E/haha: 更新状态: 4
06-05 11:33:31.193 363-363/com.questfree.duojiao E/haha: onEvent: 4
06-05 11:33:31.196 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 4
06-05 11:33:31.241 363-363/com.questfree.duojiao E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:31.269 363-363/com.questfree.duojiao E/haha: reStart: 開始任務
06-05 11:33:31.270 363-363/com.questfree.duojiao E/Download: DownloadStartEvent开始下载
06-05 11:33:31.273 363-363/com.questfree.duojiao E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.281 363-363/com.questfree.duojiao E/haha: 更新状态: 1
06-05 11:33:31.297 363-363/com.questfree.duojiao E/haha: statusToStringBybutton2: 1
06-05 11:33:31.315 363-13574/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.359 363-13531/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.385 363-11194/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.385 363-12517/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.386 363-15662/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.424 363-13531/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: update:rxPkts:2623,txPkts:2637,rxBytes:3106182,txBytes:178931
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: no expired. level:5
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: during_ms:2001,period_ms:2000
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: count:981,rx_sum:2623,tx_sum:2637,rxBytes:3106182,during_ms:2001,rx_speed:1552314.8425787108,tx_speed:89420.78960519741,rto:0.9946909366704588
06-05 11:33:31.589 363-15801/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.622 363-15801/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.669 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-60 "questfree_back_m"WPA_PSK
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: spd good count:1, add sc:0.0, rate:1.0
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: current mHighDataFlowRate = 0.0
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: POta txb txg rxg:0, 10101, 0. Lr =0% Totpkt=10101
06-05 11:33:31.674 4212-4212/? I/WifiProJNI: ######### CMD_QUERY_PKTS ########
06-05 11:33:31.675 4212-4543/? I/QosMonitor: postEventFromNative: msg=100,arg1=0,arg2=9
06-05 11:33:31.675 4212-4543/? W/WifiProJNI: poll before: g_monitor_fd =158
06-05 11:33:31.675 4212-4212/? I/MQoS: rtt=165,tcp_tx_pkts=13218009,tcp_rx_pkts=14703053
06-05 11:33:31.676 4212-4542/? I/MQoS: QOS_level:0(0),DataDisable;use level-0
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine: PTcp RTT:165, rtt pkt=319, tcp_rx=9556, tcp_tx=9468, tcp_reTran=0, rtRate=0
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine:  pkt chk not bad, reset sc to 0.
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine: net speed good
06-05 11:33:31.930 4212-4484/? I/InputDispatcher: touch process [363], send to pg
06-05 11:33:32.404 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-61 "questfree_back_m"WPA_PSK
06-05 11:33:32.714 363-363/com.questfree.duojiao I/info: SUPPORTED_64_BIT_ABIS=[Ljava.lang.String;@696fdd6
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: versionCode=1387
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: BOARD=MT7-TL00
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: BOOTLOADER=unknown
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: TYPE=user
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: matchers=[Ljava.lang.String;@9e8ef44
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: ID=HuaweiMT7-TL00
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: TIME=1474400147000
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: BRAND=Huawei
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: TAG=Build
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: SERIAL=P4M7N14902001084
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: HARDWARE=hi3630
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: SUPPORTED_ABIS=[Ljava.lang.String;@9257857
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: NO_HOTA=false
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: CPU_ABI=armeabi-v7a
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: RADIO=unknown
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: IS_DEBUGGABLE=false
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: replacements=[Ljava.lang.String;@3acc82d
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: MANUFACTURER=HUAWEI
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: SUPPORTED_32_BIT_ABIS=[Ljava.lang.String;@4b33df1
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: TAGS=release-keys
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: CPU_ABI2=armeabi
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: UNKNOWN=unknown
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: USER=jenkins
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: FINGERPRINT=Huawei/MT7-TL00/hwmt7:6.0/HuaweiMT7-TL00/C01B571:user/release-keys
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: HOST=huawei-RH2288H-V2-12L
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: PRODUCT=MT7-TL00
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: versionName=1.3.8.7
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: DISPLAY=MT7-TL00C01B571
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: HIDE_PRODUCT_INFO=false
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: MODEL=HUAWEI MT7-TL00
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: DEVICE=hwmt7
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
06-05 11:33:32.715 363-363/com.questfree.duojiao I/info:     at com.liulishuo.okdownload.core.listener.assist.Listener1Assist.connectEnd(Listener1Assist.java:88)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at com.liulishuo.okdownload.core.listener.DownloadListener1.connectEnd(DownloadListener1.java:91)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at com.liulishuo.okdownload.core.dispatcher.CallbackDispatcher$DefaultTransmitListener$7.run(CallbackDispatcher.java:297)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at android.os.Handler.handleCallback(Handler.java:743)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at android.os.Handler.dispatchMessage(Handler.java:95)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at android.os.Looper.loop(Looper.java:150)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at android.app.ActivityThread.main(ActivityThread.java:5546)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at java.lang.reflect.Method.invoke(Native Method)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
06-05 11:33:32.716 363-363/com.questfree.duojiao I/info:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
06-05 11:33:32.773 4212-4536/? E/HwCHRWebMonitor: running processNamecom.questfree.duojiao
06-05 11:33:32.774 4212-4536/? E/HwCHRWebMonitor: HwCHRWifiUIDWebSpeed [appName=com.questfree.duojiao UID=10205  WEBSENDSEGS=6018  WEBRESENDSEGS=0  WEBRECVSEGS=6189  WEBERRSEGS=0  WEBOUTRSTS=0  WEBESTABLISRST=3  WEBRTTDURATION=697  WEBRTTSEGS=166  WEBSNDDUPACKS=1  suckTimes=0 recovage_times=4]
06-05 11:33:32.775 4212-4536/? E/HwCHRWebMonitor: WebMonitor [mNetWorkIsSuck=false mUIDIsSuck=]  WebSpeed [WEBSENDSEGS=6090  WEBRESENDSEGS=0  WEBRECVSEGS=6246  WEBERRSEGS=0  WEBOUTRSTS=40  WEBESTABLISRST=3  WEBRTTDURATION=764  WEBRTTSEGS=183  WEBSRTT=165  WEBSNDDUPACKS=2  SENDSEGS=0  RESENDSEGS=0  RECVSEGS=0  ERRSEGS=0  OUTRSTS=0  ESTABLISRST=0  RTTDURATION=0  RTTSEGS=0  SNDDUPACKS=0  suckTimes=0 recovage_times=4]
06-05 11:33:33.231 363-373/com.questfree.duojiao I/art: Background partial concurrent mark sweep GC freed 291663(15MB) AllocSpace objects, 28(552KB) LOS objects, 20% free, 62MB/78MB, paused 5.363ms total 157.561ms
06-05 11:33:33.554 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:33.560 4602-4602/? I/NetworkSpeedManagerEx:  value = 1593743.0
06-05 11:33:33.563 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.5M/s
06-05 11:33:33.563 4602-4602/? I/NetworkSpeedView: /update(), speed=1.5M/s
06-05 11:33:33.572 4212-4212/? I/TrafficMonitor: update:rxPkts:2461,txPkts:2458,rxBytes:3013509,txBytes:192016
06-05 11:33:33.572 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:33.573 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:35.122 363-363/com.questfree.duojiao E/remove activity count=: 4
06-05 11:33:35.130 363-363/com.questfree.duojiao I/art: System.exit called, status: 1
06-05 11:33:35.130 363-363/com.questfree.duojiao I/AndroidRuntime: VM exiting with result code 1, cleanup skipped.
06-05 11:33:35.421 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-61 "questfree_back_m"WPA_PSK
06-05 11:33:35.438 4212-4744/? I/LocationManagerService: remove 1e2bf18
06-05 11:33:35.439 4212-4744/? I/SendBroadcastPermission: action:android.location.LOCATION_REQUEST_CHANGE_ACTION, mPermissionType:0
06-05 11:33:35.440 5538-5565/? I/PgedBinderListener: kstate callback type:8 value1=363 value2=KILLED
06-05 11:33:35.442 4212-5246/? E/HsmCoreServiceImpl: onTransact in code is: 102
06-05 11:33:35.442 4212-5246/? I/MediaProcessHandler: processOp opType: 1, uid: 10205, pid: 363
06-05 11:33:35.442 4212-5246/? W/MediaProcessHandler: remove target not exist, maybe the UI process: uid: 10205, pid: 363
06-05 11:33:35.454 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (746, 1711).
06-05 11:33:35.455 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (654, 1681).
06-05 11:33:35.456 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (563, 1654).
06-05 11:33:35.457 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (587, 1693).
06-05 11:33:35.458 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (655, 1711).
06-05 11:33:35.459 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (683, 1702).
06-05 11:33:35.461 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (649, 1703).
06-05 11:33:35.461 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (634, 1682).
06-05 11:33:35.462 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (623, 1724).
06-05 11:33:35.463 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (720, 1640).
06-05 11:33:35.463 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (683, 1668).
06-05 11:33:35.464 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (682, 1673).
06-05 11:33:35.464 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (679, 1740).
06-05 11:33:35.465 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (606, 1740).
06-05 11:33:35.466 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (535, 1726).
06-05 11:33:35.466 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (590, 1725).
06-05 11:33:35.467 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (558, 1772).
06-05 11:33:35.467 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (535, 1751).
06-05 11:33:35.500 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (567, 1748).
06-05 11:33:35.504 15816-15816/? I/art: Late-enabling -Xcheck:jni
06-05 11:33:35.538 15288-15288/? W/WifiApp: [MsgService.java,69,onTrimMemory] MsgService onTrimMemory level:10
06-05 11:33:35.541 17051-17051/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.545 11697-11697/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.576 17123-17123/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.577 4212-4212/? I/TrafficMonitor: update:rxPkts:2426,txPkts:2401,rxBytes:2979752,txBytes:181212
06-05 11:33:35.578 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:35.578 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:35.604 5599-5599/? I/HwLauncher: Launcher onStart()
06-05 11:33:35.605 5599-5599/? I/HwLauncher: Launcher dynamicIconsRegister
06-05 11:33:35.610 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.612 3664-3755/? I/[Gralloc]: alloc w[16] h[16] format[1] usage[256]
06-05 11:33:35.614 3664-3755/? I/[Gralloc]: alloc succ handle[0xb85e3de0] stride[32]
06-05 11:33:35.618 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.620 15816-15830/com.questfree.duojiao E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:35.623 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.calendar
06-05 11:33:35.624 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.629 5599-5780/? I/HwLauncher: WeatherDynamicUpdaterpositiveUpdate at WeatherDynamicUpdater
06-05 11:33:35.629 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.637 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = true   mAttachedToWindow:false    mWindowVisible:falsecom.huawei.android.totemweather
06-05 11:33:35.637 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.642 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.643 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.deskclock
06-05 11:33:35.647 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10205
06-05 11:33:35.648 5599-5599/? I/HwLauncher: Launcher  Launcher.onStart() cost 43 ms
06-05 11:33:35.656 342-399/? I/TotemWeather: Utils launcherIntent for weather DynamicIcon
06-05 11:33:35.669 3686-3686/? I/Zygote: Process 363 exited cleanly (1)
06-05 11:33:35.677 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10055
06-05 11:33:35.683 342-399/? E/TotemWeather: WidgetUtils getWidgetShowCityInfo return myLocation
06-05 11:33:35.686 342-399/? I/SendBroadcastPermission: action:com.huawei.android.action.launcher_weather_icon_change, mPermissionType:0
06-05 11:33:35.698 5599-5780/? E/HwLauncher: SettingsEx , no such field.
06-05 11:33:35.706 5599-5780/? W/HwLauncher: Clock getDataFormat the getSystemString failed.
06-05 11:33:35.711 5599-5780/? I/HwLauncher: DynamicIconData , getDrawableForDynamic begin, pkg = com.android.deskclock, picName = deskclock_bg
06-05 11:33:35.712 5599-5780/? I/HwLauncher: DynamicIconData , getDrawableForDynamic end, pkg = com.android.deskclock, picName = deskclock_bg
06-05 11:33:35.719 5599-6072/? E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:35.720 5599-6072/? I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:35.724 5599-5780/? I/HwLauncher: ClockDynamicUpdater clock update folder at ClockDynamicUpdater
06-05 11:33:35.724 5599-5780/? I/HwLauncher: DynamicUpdater updateBitmap end and send update message
06-05 11:33:35.739 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.744 3664-3755/? I/[Gralloc]: alloc succ handle[0xb85e3de0] stride[1120]
06-05 11:33:35.744 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.754 3664-3755/? I/[Gralloc]: alloc succ handle[0xb86040c8] stride[1120]
06-05 11:33:35.754 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.760 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent begin, false, true] down event  x = 531 y = 531
06-05 11:33:35.763 3664-3755/? I/[Gralloc]: alloc succ handle[0xb84e00a0] stride[1120]
06-05 11:33:35.764 5599-5599/? I/HwLauncher: Launcher  while lockScreenOrientation,isRotationEnabled is false!
06-05 11:33:35.790 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent end, false, true] down event  x = 531 y = 531
06-05 11:33:35.791 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent begin, false, true] up event  x = 531 y = 531
06-05 11:33:35.792 5599-5599/? I/HwLauncher: QuickActionView hide() animated = true, fromSticky = true
06-05 11:33:35.792 5599-5599/? I/HwLauncher: QuickActionView hide() action view has already hidden, return.
06-05 11:33:35.797 5599-5599/? I/HwLauncher: Launcher mUnlockScreenRunnbale  getOrientationEnabled false 
06-05 11:33:35.797 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent end, false, true] up event  x = 531 y = 531
06-05 11:33:35.801 15816-15816/com.questfree.duojiao I/MultiDex: VM with version 2.1.0 has multidex support
06-05 11:33:35.804 15816-15816/com.questfree.duojiao I/MultiDex: Installing application
06-05 11:33:35.804 15816-15816/com.questfree.duojiao I/MultiDex: VM has multidex support, MultiDex support library is disabled.
06-05 11:33:35.808 5599-5599/? I/HwLauncher: WeatherDynamicUpdaterreceive action = com.huawei.android.action.launcher_weather_icon_change
06-05 11:33:35.809 5599-5780/? I/HwLauncher: WeatherDynamicUpdaterreceiver data is int[] = [0, 0, 72, 0, 96, 0, 120, 0]
                                              stringArray[] [rain, num1, num9, tempc]
06-05 11:33:35.915 15816-15816/com.questfree.duojiao W/ImageLoader: memoryCache() and memoryCacheSize() calls overlap each other
06-05 11:33:35.915 15816-15816/com.questfree.duojiao W/ImageLoader: memoryCache() and memoryCacheSize() calls overlap each other
06-05 11:33:35.928 15816-15816/com.questfree.duojiao W/ImageLoader: diskCache(), diskCacheSize() and diskCacheFileCount calls overlap each other
06-05 11:33:35.928 15816-15816/com.questfree.duojiao W/ImageLoader: diskCache(), diskCacheSize() and diskCacheFileCount calls overlap each other
06-05 11:33:35.928 15816-15816/com.questfree.duojiao W/ImageLoader: diskCache() and diskCacheFileNameGenerator() calls overlap each other
06-05 11:33:36.011 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:36.014 5599-5599/? W/ResourceType: No known package when getting name for resource number 0x9b020200
06-05 11:33:36.014 5599-5599/? I/HwLauncher: Launcher onclick begin com.huawei.android.launcher.BubbleTextView{a6888eb VFED..CL. ...P.... 422,0-633,273 #9b020200}
06-05 11:33:36.022 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely useLaunchAnimation = true
06-05 11:33:36.024 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely begin startActivity() for current user
06-05 11:33:36.044 5538-5538/? I/K3V3CpuGovernorPolicy: set scene Hmp policy : 300 , 150
06-05 11:33:36.046 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely end startActivity() for current user
06-05 11:33:36.048 5599-5599/? I/BD.Reporter: com.huawei.bd.IBDService$Stub$Proxy@eb578a7
06-05 11:33:36.052 5599-5599/? I/HwLauncher: Launcher onclick end
06-05 11:33:36.171 15816-15816/com.questfree.duojiao W/linker: /data/app/com.questfree.duojiao-1/lib/arm/libutility.so has text relocations. This is wasting memory and prevents security hardening. Please fix.
06-05 11:33:36.193 15816-15816/com.questfree.duojiao I/[VCamera]: Application package name: com.questfree.duojiao
06-05 11:33:36.193 15816-15816/com.questfree.duojiao I/[VCamera]: Copyright (c) YIXIA (http://miaopai.yixia.com)
06-05 11:33:36.242 4212-4251/? I/Bluetooth_framework: BluetoothManagerService:Message: 20
06-05 11:33:36.242 4212-4251/? I/Bluetooth_framework: BluetoothManagerService:Added callback: android.bluetooth.IBluetoothManagerCallback$Stub$Proxy@c38275e:true pid = 15816
06-05 11:33:36.320 15816-15816/com.questfree.duojiao I/HwCust: Constructor found for class android.app.HwCustHwWallpaperManagerImpl
06-05 11:33:36.328 5554-6170/? W/PhoneInterfaceManager: shouldBlockLocation running ...
06-05 11:33:36.340 5554-6170/? I/PhoneInterfaceManager: shouldBlockLocation  ret:false
06-05 11:33:36.348 15816-15816/com.questfree.duojiao E/com.questfree.duojiao: >>>>>>>>>>>>>>>>>>>切到前台 activity process
06-05 11:33:36.353 5599-6072/? W/OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
06-05 11:33:36.353 5599-6072/? W/OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
06-05 11:33:36.395 15816-15872/com.questfree.duojiao I/System.out: [/system/xbin/which, su]
06-05 11:33:36.395 15816-15872/com.questfree.duojiao I/System.out: null
06-05 11:33:36.395 15816-15872/com.questfree.duojiao I/System.out: null
06-05 11:33:36.400 15816-15872/com.questfree.duojiao I/System.out: Calling by::className:com.umeng.commonsdk.internal.utils.e  MethodName:a
06-05 11:33:36.435 5538-6472/? W/PGApi_client: recv actoionId = 10013, action = com.huawei.pgmng.PGAction@ff71ced actionId =10013 pkg =com.android.mms extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.438 5538-6472/? W/PGMiddleWare: in handleAction method, action = 10013
06-05 11:33:36.438 5538-6472/? W/PGMiddleWare: in handleAction, invoke client = com.huawei.pgmng.middleware.AudioEffectLowPowerImpl@f7e3997, action = com.huawei.pgmng.PGAction@ff71ced actionId =10013 pkg =com.android.mms extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.444 5599-5599/? I/HwLauncher: Launcher onStop()
06-05 11:33:36.463 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10031
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: expired arrive. level:-1
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: during_ms:2006,period_ms:2000
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: count:983,rx_sum:2426,tx_sum:2401,rxBytes:2979752,during_ms:2006,rx_speed:1485419.7407776671,tx_speed:90334.99501495514,rto:1.0104123281965847
06-05 11:33:36.566 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:36.567 15816-15877/com.questfree.duojiao E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:36.567 15816-15877/com.questfree.duojiao I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:36.571 4602-4602/? I/NetworkSpeedManagerEx:  value = 1122148.4
06-05 11:33:36.572 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.1M/s
06-05 11:33:36.572 4602-4602/? I/NetworkSpeedView: /update(), speed=1.1M/s
06-05 11:33:36.614 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.615 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.615 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.627 20164-20164/? E/OsUtil: Buggle checkHasSmsPermissionsForUser Enable 10031
06-05 11:33:36.651 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.653 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.653 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.666 20164-20213/? E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:36.667 20164-20213/? I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:36.679 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.681 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.682 15816-15872/com.questfree.duojiao I/CameraManagerGlobal: Connecting to camera service
06-05 11:33:36.695 5538-6472/? W/PGApi_client: recv actoionId = 10000, action = com.huawei.pgmng.PGAction@355b722 actionId =10000 pkg =com.questfree.duojiao extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.696 5538-6472/? W/PGMiddleWare: in handleAction method, action = 10000
06-05 11:33:36.696 5538-6472/? W/PGMiddleWare: in handleAction, invoke client = com.huawei.pgmng.middleware.AudioEffectLowPowerImpl@f7e3997, action = com.huawei.pgmng.PGAction@355b722 actionId =10000 pkg =com.questfree.duojiao extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.703 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10205
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.709 3809-3809/? I/CameraService: getCameraCharacteristics: Switching to HAL1 shim implementation...
06-05 11:33:36.710 3809-3942/? I/CameraService: getCameraCharacteristics: Switching to HAL1 shim implementation...
06-05 11:33:36.717 3664-3757/? I/[Gralloc]: alloc succ handle[0xb85692d8] stride[1120]
06-05 11:33:36.717 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.728 3664-5199/? I/[Gralloc]: alloc succ handle[0xb8568ed8] stride[1120]
06-05 11:33:36.728 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.740 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.742 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.743 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.763 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.764 3664-3757/? I/[Gralloc]: alloc succ handle[0xb856ac78] stride[1120]
06-05 11:33:36.764 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.764 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.764 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.784 20164-20164/? E/Mms/cache: getSingleContact error for: 
06-05 11:33:36.785 20164-20164/? E/Mms: [1] Mms_conv
06-05 11:33:36.788 3664-5199/? I/[Gralloc]: alloc succ handle[0xb85bac20] stride[1120]
06-05 11:33:36.788 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.823 3664-3757/? I/[Gralloc]: alloc succ handle[0xb8606d88] stride[1120]
06-05 11:33:36.837 3664-5199/? I/[Gralloc]: alloc succ handle[0xb8594f40] stride[1120]
06-05 11:33:36.846 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.870 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84378f8] stride[1120]
06-05 11:33:36.870 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.891 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84dee18] stride[1120]
06-05 11:33:36.891 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.910 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84ded38] stride[1120]
06-05 11:33:37.050 15816-15896/com.questfree.duojiao I/System.out: true
06-05 11:33:37.077 15816-15877/com.questfree.duojiao E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:37.077 15816-15877/com.questfree.duojiao I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:37.086 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.086 15816-15896/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.117 15816-15896/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.125 3664-3755/? I/[Gralloc]: alloc succ handle[0xb8602d30] stride[1120]
06-05 11:33:37.125 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.151 15816-15899/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.152 3664-3755/? I/[Gralloc]: alloc succ handle[0xb86150a8] stride[1120]
06-05 11:33:37.152 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.174 3664-3755/? I/[Gralloc]: alloc succ handle[0xb8586db0] stride[1120]
06-05 11:33:37.184 15816-15899/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.233 15816-15899/com.questfree.duojiao W/System.err: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.240 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:328)
06-05 11:33:37.240 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.http.SocketConnector.connectTls(SocketConnector.java:103)
06-05 11:33:37.240 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.Connection.connect(Connection.java:143)
06-05 11:33:37.241 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.Connection.connectAndSetOwner(Connection.java:185)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.OkHttpClient$1.connectAndSetOwner(OkHttpClient.java:128)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.http.HttpEngine.nextConnection(HttpEngine.java:341)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:330)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:248)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:433)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect(HttpURLConnectionImpl.java:114)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getOutputStream(HttpURLConnectionImpl.java:245)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getOutputStream(DelegatingHttpsURLConnection.java:218)
06-05 11:33:37.242 15816-15899/com.questfree.duojiao W/System.err:     at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:25)
06-05 11:33:37.249 15816-15899/com.questfree.duojiao W/System.err:     at cn.jpush.android.util.s.a(Unknown Source)
06-05 11:33:37.249 15816-15899/com.questfree.duojiao W/System.err:     at cn.jpush.android.util.aj.b(Unknown Source)
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err:     at cn.jpush.android.util.al.run(Unknown Source)
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err:     at java.lang.Thread.run(Thread.java:833)
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err: Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.250 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:324)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.TrustManagerImpl.checkServerTrusted(TrustManagerImpl.java:215)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at cn.jpush.android.util.u.checkServerTrusted(Unknown Source)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.Platform.checkServerTrusted(Platform.java:117)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.verifyCertificateChain(OpenSSLSocketImpl.java:556)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.NativeCrypto.SSL_do_handshake(Native Method)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:324)
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err: 	... 18 more
06-05 11:33:37.251 15816-15899/com.questfree.duojiao W/System.err: Caused by: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.253 15816-15899/com.questfree.duojiao W/System.err: 	... 25 more
06-05 11:33:37.263 4212-4252/? I/ActivityManager: Displayed com.questfree.duojiao/.t4.android.ThinksnsActivity: +578ms (total +1s790ms)
06-05 11:33:37.272 15816-15899/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.303 15816-15899/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.411 5538-5538/? I/K3V3CpuGovernorPolicy: close previously scene Hmp policy : 300 , 150
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: update:rxPkts:193,txPkts:220,rxBytes:176530,txBytes:17968
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: no expired. level:5
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:37.580 4212-4212/? I/TrafficMonitor: during_ms:2002,period_ms:2000
06-05 11:33:37.580 4212-4212/? I/TrafficMonitor: count:984,rx_sum:193,tx_sum:220,rxBytes:176530,during_ms:2002,rx_speed:88176.82317682319,tx_speed:8975.024975024977,rto:0.8772727272727273
06-05 11:33:37.727 15816-15873/com.questfree.duojiao I/System.out: true
06-05 11:33:37.752 15816-15873/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.786 15816-15873/com.questfree.duojiao I/System: core_booster, getBoosterConfig = false`

06-05 11:33:29.448 363-363/myPackage E/haha: reStart: 開始任務
06-05 11:33:29.449 363-363/myPackage E/Download: DownloadStartEvent开始下载
06-05 11:33:29.449 15488-15652/? I/System.out: [getprop, ro.product.cpu.abi]
06-05 11:33:29.449 15488-15652/? I/System.out: null
06-05 11:33:29.449 15488-15652/? I/System.out: null
06-05 11:33:29.450 15488-15652/? I/System.out: Calling by::className:com.tencent.tbs.common.utils.DeviceUtils  MethodName:getDeviceCpuabi
06-05 11:33:29.453 363-363/myPackage E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.473 363-363/myPackage E/haha: 更新状态: 1
06-05 11:33:29.487 363-363/myPackage E/haha: statusToStringBybutton2: 1
06-05 11:33:29.487 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.499 363-12503/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.557 363-11194/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.557 4212-4212/? I/TrafficMonitor: update:rxPkts:2494,txPkts:2363,rxBytes:2961714,txBytes:153181
06-05 11:33:29.557 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:29.558 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:29.567 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.588 363-15662/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.588 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.590 363-13327/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.602 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.612 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:29.613 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.627 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.658 363-11194/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.668 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:29.669 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.681 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:29.681 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.689 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:29.694 363-363/myPackage E/haha: STATUS_CONNECTED
06-05 11:33:29.696 363-363/myPackage E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:29.697 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.709 363-363/myPackage E/QueueController: taskEnd: CANCELED
06-05 11:33:29.709 363-363/myPackage E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:29.709 363-363/myPackage E/haha: 更新状态: 4
06-05 11:33:29.747 363-363/myPackage E/haha: onEvent: 4
06-05 11:33:29.749 363-363/myPackage E/haha: statusToStringBybutton2: 4
06-05 11:33:29.785 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.889 363-363/myPackage E/haha: reStart: 開始任務
06-05 11:33:29.893 363-363/myPackage E/Download: DownloadStartEvent开始下载
06-05 11:33:29.896 363-363/myPackage E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:29.897 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:29.911 363-363/myPackage E/haha: 更新状态: 1
06-05 11:33:29.925 363-363/myPackage E/haha: statusToStringBybutton2: 1
06-05 11:33:29.939 363-12503/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:29.972 363-15662/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.005 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.007 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.008 363-11194/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.010 363-13327/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.019 363-15662/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.025 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.047 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.049 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.056 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.069 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.070 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.080 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.080 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.092 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.096 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.108 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.117 363-363/myPackage E/haha: STATUS_CONNECTED
06-05 11:33:30.118 363-363/myPackage E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:30.139 363-363/myPackage E/QueueController: taskEnd: CANCELED
06-05 11:33:30.139 363-363/myPackage E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:30.139 363-363/myPackage E/haha: 更新状态: 4
06-05 11:33:30.161 363-363/myPackage E/haha: onEvent: 4
06-05 11:33:30.163 363-363/myPackage E/haha: statusToStringBybutton2: 4
06-05 11:33:30.191 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.306 363-363/myPackage E/haha: reStart: 開始任務
06-05 11:33:30.307 363-363/myPackage E/Download: DownloadStartEvent开始下载
06-05 11:33:30.310 363-363/myPackage E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.320 363-363/myPackage E/haha: 更新状态: 1
06-05 11:33:30.332 363-363/myPackage E/haha: statusToStringBybutton2: 1
06-05 11:33:30.332 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.352 363-12503/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.388 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.416 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.431 363-13531/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.434 363-15662/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.434 363-13327/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.439 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.455 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.456 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.456 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:30.472 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.501 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.501 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.515 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.516 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.528 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.531 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.541 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.548 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:30.550 4602-4602/? I/NetworkSpeedManagerEx:  value = 1601218.6
06-05 11:33:30.551 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.5M/s
06-05 11:33:30.551 4602-4602/? I/NetworkSpeedView: /update(), speed=1.5M/s
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: expired arrive. level:-1
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: during_ms:2002,period_ms:2000
06-05 11:33:30.553 4212-4212/? I/TrafficMonitor: count:980,rx_sum:2494,tx_sum:2363,rxBytes:2961714,during_ms:2002,rx_speed:1479377.6223776226,tx_speed:76513.98601398602,rto:1.0554380025391452
06-05 11:33:30.562 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:30.563 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.641 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.658 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.708 363-363/myPackage E/haha: STATUS_CONNECTED
06-05 11:33:30.709 363-363/myPackage E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:30.721 363-363/myPackage E/QueueController: taskEnd: CANCELED
06-05 11:33:30.722 363-363/myPackage E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:30.722 363-363/myPackage E/haha: 更新状态: 4
06-05 11:33:30.760 363-363/myPackage E/haha: onEvent: 4
06-05 11:33:30.762 363-363/myPackage E/haha: statusToStringBybutton2: 4
06-05 11:33:30.852 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.903 363-363/myPackage E/haha: reStart: 開始任務
06-05 11:33:30.904 363-363/myPackage E/Download: DownloadStartEvent开始下载
06-05 11:33:30.907 363-363/myPackage E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:30.911 363-363/myPackage E/haha: 更新状态: 1
06-05 11:33:30.930 363-363/myPackage E/haha: statusToStringBybutton2: 1
06-05 11:33:30.946 363-373/myPackage I/art: Background sticky concurrent mark sweep GC freed 293629(15MB) AllocSpace objects, 25(500KB) LOS objects, 19% free, 62MB/78MB, paused 8.949ms total 133.886ms
06-05 11:33:30.953 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:30.960 363-12503/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.003 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.030 363-11194/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.031 363-13327/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.035 363-13531/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.046 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.048 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.065 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:31.066 363-363/myPackage E/QueueController: progress: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.077 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.092 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:31.096 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.114 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:31.116 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:31.122 363-363/myPackage E/QueueController: connected: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.141 363-363/myPackage E/haha: statusToStringBybutton2: 5
06-05 11:33:31.145 363-363/myPackage E/haha: STATUS_CONNECTED
06-05 11:33:31.146 363-363/myPackage E/haha: pauseSingleTask: 任務開始暂停
06-05 11:33:31.167 363-363/myPackage E/QueueController: taskEnd: CANCELED
06-05 11:33:31.168 363-363/myPackage E/haha: pauseSingleTask: 任務暫停成功回掉
06-05 11:33:31.168 363-363/myPackage E/haha: 更新状态: 4
06-05 11:33:31.193 363-363/myPackage E/haha: onEvent: 4
06-05 11:33:31.196 363-363/myPackage E/haha: statusToStringBybutton2: 4
06-05 11:33:31.241 363-363/myPackage E/QueueController: progress: whcl2_44_9.apk
06-05 11:33:31.269 363-363/myPackage E/haha: reStart: 開始任務
06-05 11:33:31.270 363-363/myPackage E/Download: DownloadStartEvent开始下载
06-05 11:33:31.273 363-363/myPackage E/QueueController: RUNNING: f66bdf52a135a25bd86ca8f42a69e1b4_11625.apk
06-05 11:33:31.281 363-363/myPackage E/haha: 更新状态: 1
06-05 11:33:31.297 363-363/myPackage E/haha: statusToStringBybutton2: 1
06-05 11:33:31.315 363-13574/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.359 363-13531/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.385 363-11194/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.385 363-12517/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.386 363-15662/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.424 363-13531/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: update:rxPkts:2623,txPkts:2637,rxBytes:3106182,txBytes:178931
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: no expired. level:5
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: during_ms:2001,period_ms:2000
06-05 11:33:31.559 4212-4212/? I/TrafficMonitor: count:981,rx_sum:2623,tx_sum:2637,rxBytes:3106182,during_ms:2001,rx_speed:1552314.8425787108,tx_speed:89420.78960519741,rto:0.9946909366704588
06-05 11:33:31.589 363-15801/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.622 363-15801/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:31.669 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-60 "questfree_back_m"WPA_PSK
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: spd good count:1, add sc:0.0, rate:1.0
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: current mHighDataFlowRate = 0.0
06-05 11:33:31.674 4212-4545/? I/HuaweiWifiWatchdogStateMachine: POta txb txg rxg:0, 10101, 0. Lr =0% Totpkt=10101
06-05 11:33:31.674 4212-4212/? I/WifiProJNI: ######### CMD_QUERY_PKTS ########
06-05 11:33:31.675 4212-4543/? I/QosMonitor: postEventFromNative: msg=100,arg1=0,arg2=9
06-05 11:33:31.675 4212-4543/? W/WifiProJNI: poll before: g_monitor_fd =158
06-05 11:33:31.675 4212-4212/? I/MQoS: rtt=165,tcp_tx_pkts=13218009,tcp_rx_pkts=14703053
06-05 11:33:31.676 4212-4542/? I/MQoS: QOS_level:0(0),DataDisable;use level-0
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine: PTcp RTT:165, rtt pkt=319, tcp_rx=9556, tcp_tx=9468, tcp_reTran=0, rtRate=0
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine:  pkt chk not bad, reset sc to 0.
06-05 11:33:31.677 4212-4545/? I/HuaweiWifiWatchdogStateMachine: net speed good
06-05 11:33:31.930 4212-4484/? I/InputDispatcher: touch process [363], send to pg
06-05 11:33:32.404 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-61 "questfree_back_m"WPA_PSK
06-05 11:33:32.714 363-363/myPackage I/info: SUPPORTED_64_BIT_ABIS=[Ljava.lang.String;@696fdd6
06-05 11:33:32.715 363-363/myPackage I/info: versionCode=1387
06-05 11:33:32.715 363-363/myPackage I/info: BOARD=MT7-TL00
06-05 11:33:32.715 363-363/myPackage I/info: BOOTLOADER=unknown
06-05 11:33:32.715 363-363/myPackage I/info: TYPE=user
06-05 11:33:32.715 363-363/myPackage I/info: matchers=[Ljava.lang.String;@9e8ef44
06-05 11:33:32.715 363-363/myPackage I/info: ID=HuaweiMT7-TL00
06-05 11:33:32.715 363-363/myPackage I/info: TIME=1474400147000
06-05 11:33:32.715 363-363/myPackage I/info: BRAND=Huawei
06-05 11:33:32.715 363-363/myPackage I/info: TAG=Build
06-05 11:33:32.715 363-363/myPackage I/info: SERIAL=P4M7N14902001084
06-05 11:33:32.715 363-363/myPackage I/info: HARDWARE=hi3630
06-05 11:33:32.715 363-363/myPackage I/info: SUPPORTED_ABIS=[Ljava.lang.String;@9257857
06-05 11:33:32.715 363-363/myPackage I/info: NO_HOTA=false
06-05 11:33:32.715 363-363/myPackage I/info: CPU_ABI=armeabi-v7a
06-05 11:33:32.715 363-363/myPackage I/info: RADIO=unknown
06-05 11:33:32.715 363-363/myPackage I/info: IS_DEBUGGABLE=false
06-05 11:33:32.715 363-363/myPackage I/info: replacements=[Ljava.lang.String;@3acc82d
06-05 11:33:32.715 363-363/myPackage I/info: MANUFACTURER=HUAWEI
06-05 11:33:32.715 363-363/myPackage I/info: SUPPORTED_32_BIT_ABIS=[Ljava.lang.String;@4b33df1
06-05 11:33:32.715 363-363/myPackage I/info: TAGS=release-keys
06-05 11:33:32.715 363-363/myPackage I/info: CPU_ABI2=armeabi
06-05 11:33:32.715 363-363/myPackage I/info: UNKNOWN=unknown
06-05 11:33:32.715 363-363/myPackage I/info: USER=jenkins
06-05 11:33:32.715 363-363/myPackage I/info: FINGERPRINT=Huawei/MT7-TL00/hwmt7:6.0/HuaweiMT7-TL00/C01B571:user/release-keys
06-05 11:33:32.715 363-363/myPackage I/info: HOST=huawei-RH2288H-V2-12L
06-05 11:33:32.715 363-363/myPackage I/info: PRODUCT=MT7-TL00
06-05 11:33:32.715 363-363/myPackage I/info: versionName=1.3.8.7
06-05 11:33:32.715 363-363/myPackage I/info: DISPLAY=MT7-TL00C01B571
06-05 11:33:32.715 363-363/myPackage I/info: HIDE_PRODUCT_INFO=false
06-05 11:33:32.715 363-363/myPackage I/info: MODEL=HUAWEI MT7-TL00
06-05 11:33:32.715 363-363/myPackage I/info: DEVICE=hwmt7
06-05 11:33:32.715 363-363/myPackage I/info: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
06-05 11:33:32.715 363-363/myPackage I/info:     at com.liulishuo.okdownload.core.listener.assist.Listener1Assist.connectEnd(Listener1Assist.java:88)
06-05 11:33:32.716 363-363/myPackage I/info:     at com.liulishuo.okdownload.core.listener.DownloadListener1.connectEnd(DownloadListener1.java:91)
06-05 11:33:32.716 363-363/myPackage I/info:     at com.liulishuo.okdownload.core.dispatcher.CallbackDispatcher$DefaultTransmitListener$7.run(CallbackDispatcher.java:297)
06-05 11:33:32.716 363-363/myPackage I/info:     at android.os.Handler.handleCallback(Handler.java:743)
06-05 11:33:32.716 363-363/myPackage I/info:     at android.os.Handler.dispatchMessage(Handler.java:95)
06-05 11:33:32.716 363-363/myPackage I/info:     at android.os.Looper.loop(Looper.java:150)
06-05 11:33:32.716 363-363/myPackage I/info:     at android.app.ActivityThread.main(ActivityThread.java:5546)
06-05 11:33:32.716 363-363/myPackage I/info:     at java.lang.reflect.Method.invoke(Native Method)
06-05 11:33:32.716 363-363/myPackage I/info:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
06-05 11:33:32.716 363-363/myPackage I/info:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
06-05 11:33:32.773 4212-4536/? E/HwCHRWebMonitor: running processNamemyPackage
06-05 11:33:32.774 4212-4536/? E/HwCHRWebMonitor: HwCHRWifiUIDWebSpeed [appName=myPackage UID=10205  WEBSENDSEGS=6018  WEBRESENDSEGS=0  WEBRECVSEGS=6189  WEBERRSEGS=0  WEBOUTRSTS=0  WEBESTABLISRST=3  WEBRTTDURATION=697  WEBRTTSEGS=166  WEBSNDDUPACKS=1  suckTimes=0 recovage_times=4]
06-05 11:33:32.775 4212-4536/? E/HwCHRWebMonitor: WebMonitor [mNetWorkIsSuck=false mUIDIsSuck=]  WebSpeed [WEBSENDSEGS=6090  WEBRESENDSEGS=0  WEBRECVSEGS=6246  WEBERRSEGS=0  WEBOUTRSTS=40  WEBESTABLISRST=3  WEBRTTDURATION=764  WEBRTTSEGS=183  WEBSRTT=165  WEBSNDDUPACKS=2  SENDSEGS=0  RESENDSEGS=0  RECVSEGS=0  ERRSEGS=0  OUTRSTS=0  ESTABLISRST=0  RTTDURATION=0  RTTSEGS=0  SNDDUPACKS=0  suckTimes=0 recovage_times=4]
06-05 11:33:33.231 363-373/myPackage I/art: Background partial concurrent mark sweep GC freed 291663(15MB) AllocSpace objects, 28(552KB) LOS objects, 20% free, 62MB/78MB, paused 5.363ms total 157.561ms
06-05 11:33:33.554 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:33.560 4602-4602/? I/NetworkSpeedManagerEx:  value = 1593743.0
06-05 11:33:33.563 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.5M/s
06-05 11:33:33.563 4602-4602/? I/NetworkSpeedView: /update(), speed=1.5M/s
06-05 11:33:33.572 4212-4212/? I/TrafficMonitor: update:rxPkts:2461,txPkts:2458,rxBytes:3013509,txBytes:192016
06-05 11:33:33.572 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:33.573 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:35.122 363-363/myPackage E/remove activity count=: 4
06-05 11:33:35.130 363-363/myPackage I/art: System.exit called, status: 1
06-05 11:33:35.130 363-363/myPackage I/AndroidRuntime: VM exiting with result code 1, cleanup skipped.
06-05 11:33:35.421 4212-4503/? E/WifiConfigStore: updateConfiguration freq=5220 BSSID=8c:a6:df:ca:3e:df RSSI=-61 "questfree_back_m"WPA_PSK
06-05 11:33:35.438 4212-4744/? I/LocationManagerService: remove 1e2bf18
06-05 11:33:35.439 4212-4744/? I/SendBroadcastPermission: action:android.location.LOCATION_REQUEST_CHANGE_ACTION, mPermissionType:0
06-05 11:33:35.440 5538-5565/? I/PgedBinderListener: kstate callback type:8 value1=363 value2=KILLED
06-05 11:33:35.442 4212-5246/? E/HsmCoreServiceImpl: onTransact in code is: 102
06-05 11:33:35.442 4212-5246/? I/MediaProcessHandler: processOp opType: 1, uid: 10205, pid: 363
06-05 11:33:35.442 4212-5246/? W/MediaProcessHandler: remove target not exist, maybe the UI process: uid: 10205, pid: 363
06-05 11:33:35.454 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (746, 1711).
06-05 11:33:35.455 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (654, 1681).
06-05 11:33:35.456 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (563, 1654).
06-05 11:33:35.457 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (587, 1693).
06-05 11:33:35.458 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (655, 1711).
06-05 11:33:35.459 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (683, 1702).
06-05 11:33:35.461 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (649, 1703).
06-05 11:33:35.461 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (634, 1682).
06-05 11:33:35.462 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (623, 1724).
06-05 11:33:35.463 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (720, 1640).
06-05 11:33:35.463 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (683, 1668).
06-05 11:33:35.464 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (682, 1673).
06-05 11:33:35.464 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (679, 1740).
06-05 11:33:35.465 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (606, 1740).
06-05 11:33:35.466 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (535, 1726).
06-05 11:33:35.466 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (590, 1725).
06-05 11:33:35.467 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (558, 1772).
06-05 11:33:35.467 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (535, 1751).
06-05 11:33:35.500 4212-4484/? I/InputDispatcher: Dropping event because there is no touchable window at (567, 1748).
06-05 11:33:35.504 15816-15816/? I/art: Late-enabling -Xcheck:jni
06-05 11:33:35.538 15288-15288/? W/WifiApp: [MsgService.java,69,onTrimMemory] MsgService onTrimMemory level:10
06-05 11:33:35.541 17051-17051/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.545 11697-11697/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.576 17123-17123/? W/WifiApp: [Configuration.java,3289,a] conf is Null: null
06-05 11:33:35.577 4212-4212/? I/TrafficMonitor: update:rxPkts:2426,txPkts:2401,rxBytes:2979752,txBytes:181212
06-05 11:33:35.578 4212-4212/? I/TrafficMonitor: start expired. level:5
06-05 11:33:35.578 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:35.604 5599-5599/? I/HwLauncher: Launcher onStart()
06-05 11:33:35.605 5599-5599/? I/HwLauncher: Launcher dynamicIconsRegister
06-05 11:33:35.610 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.612 3664-3755/? I/[Gralloc]: alloc w[16] h[16] format[1] usage[256]
06-05 11:33:35.614 3664-3755/? I/[Gralloc]: alloc succ handle[0xb85e3de0] stride[32]
06-05 11:33:35.618 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.620 15816-15830/myPackage E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:35.623 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.calendar
06-05 11:33:35.624 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.629 5599-5780/? I/HwLauncher: WeatherDynamicUpdaterpositiveUpdate at WeatherDynamicUpdater
06-05 11:33:35.629 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.637 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = true   mAttachedToWindow:false    mWindowVisible:falsecom.huawei.android.totemweather
06-05 11:33:35.637 5599-5599/? I/HwLauncher: DynamicUpdater registerReceiver
06-05 11:33:35.642 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:35.643 5599-5599/? I/HwLauncher: DynamicIcon onResume  isvisible = false   mAttachedToWindow:true    mWindowVisible:falsecom.android.deskclock
06-05 11:33:35.647 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10205
06-05 11:33:35.648 5599-5599/? I/HwLauncher: Launcher  Launcher.onStart() cost 43 ms
06-05 11:33:35.656 342-399/? I/TotemWeather: Utils launcherIntent for weather DynamicIcon
06-05 11:33:35.669 3686-3686/? I/Zygote: Process 363 exited cleanly (1)
06-05 11:33:35.677 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10055
06-05 11:33:35.683 342-399/? E/TotemWeather: WidgetUtils getWidgetShowCityInfo return myLocation
06-05 11:33:35.686 342-399/? I/SendBroadcastPermission: action:com.huawei.android.action.launcher_weather_icon_change, mPermissionType:0
06-05 11:33:35.698 5599-5780/? E/HwLauncher: SettingsEx , no such field.
06-05 11:33:35.706 5599-5780/? W/HwLauncher: Clock getDataFormat the getSystemString failed.
06-05 11:33:35.711 5599-5780/? I/HwLauncher: DynamicIconData , getDrawableForDynamic begin, pkg = com.android.deskclock, picName = deskclock_bg
06-05 11:33:35.712 5599-5780/? I/HwLauncher: DynamicIconData , getDrawableForDynamic end, pkg = com.android.deskclock, picName = deskclock_bg
06-05 11:33:35.719 5599-6072/? E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:35.720 5599-6072/? I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:35.724 5599-5780/? I/HwLauncher: ClockDynamicUpdater clock update folder at ClockDynamicUpdater
06-05 11:33:35.724 5599-5780/? I/HwLauncher: DynamicUpdater updateBitmap end and send update message
06-05 11:33:35.739 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.744 3664-3755/? I/[Gralloc]: alloc succ handle[0xb85e3de0] stride[1120]
06-05 11:33:35.744 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.754 3664-3755/? I/[Gralloc]: alloc succ handle[0xb86040c8] stride[1120]
06-05 11:33:35.754 3664-3755/? I/[Gralloc]: alloc w[1080] h[1920] format[1] usage[2816]
06-05 11:33:35.760 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent begin, false, true] down event  x = 531 y = 531
06-05 11:33:35.763 3664-3755/? I/[Gralloc]: alloc succ handle[0xb84e00a0] stride[1120]
06-05 11:33:35.764 5599-5599/? I/HwLauncher: Launcher  while lockScreenOrientation,isRotationEnabled is false!
06-05 11:33:35.790 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent end, false, true] down event  x = 531 y = 531
06-05 11:33:35.791 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent begin, false, true] up event  x = 531 y = 531
06-05 11:33:35.792 5599-5599/? I/HwLauncher: QuickActionView hide() animated = true, fromSticky = true
06-05 11:33:35.792 5599-5599/? I/HwLauncher: QuickActionView hide() action view has already hidden, return.
06-05 11:33:35.797 5599-5599/? I/HwLauncher: Launcher mUnlockScreenRunnbale  getOrientationEnabled false 
06-05 11:33:35.797 5599-5599/? I/HwLauncher:  FPSMonitor [Launcher , dispatchTouchEvent end, false, true] up event  x = 531 y = 531
06-05 11:33:35.801 15816-15816/myPackage I/MultiDex: VM with version 2.1.0 has multidex support
06-05 11:33:35.804 15816-15816/myPackage I/MultiDex: Installing application
06-05 11:33:35.804 15816-15816/myPackage I/MultiDex: VM has multidex support, MultiDex support library is disabled.
06-05 11:33:35.808 5599-5599/? I/HwLauncher: WeatherDynamicUpdaterreceive action = com.huawei.android.action.launcher_weather_icon_change
06-05 11:33:35.809 5599-5780/? I/HwLauncher: WeatherDynamicUpdaterreceiver data is int[] = [0, 0, 72, 0, 96, 0, 120, 0]
                                              stringArray[] [rain, num1, num9, tempc]
06-05 11:33:35.915 15816-15816/myPackage W/ImageLoader: memoryCache() and memoryCacheSize() calls overlap each other
06-05 11:33:35.915 15816-15816/myPackage W/ImageLoader: memoryCache() and memoryCacheSize() calls overlap each other
06-05 11:33:35.928 15816-15816/myPackage W/ImageLoader: diskCache(), diskCacheSize() and diskCacheFileCount calls overlap each other
06-05 11:33:35.928 15816-15816/myPackage W/ImageLoader: diskCache(), diskCacheSize() and diskCacheFileCount calls overlap each other
06-05 11:33:35.928 15816-15816/myPackage W/ImageLoader: diskCache() and diskCacheFileNameGenerator() calls overlap each other
06-05 11:33:36.011 5599-5599/? I/HwLauncher: DynamicUpdater call updateFolder
06-05 11:33:36.014 5599-5599/? W/ResourceType: No known package when getting name for resource number 0x9b020200
06-05 11:33:36.014 5599-5599/? I/HwLauncher: Launcher onclick begin com.huawei.android.launcher.BubbleTextView{a6888eb VFED..CL. ...P.... 422,0-633,273 #9b020200}
06-05 11:33:36.022 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely useLaunchAnimation = true
06-05 11:33:36.024 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely begin startActivity() for current user
06-05 11:33:36.044 5538-5538/? I/K3V3CpuGovernorPolicy: set scene Hmp policy : 300 , 150
06-05 11:33:36.046 5599-5599/? I/HwLauncher: Launcher.Utilities  startActivitySafely end startActivity() for current user
06-05 11:33:36.048 5599-5599/? I/BD.Reporter: com.huawei.bd.IBDService$Stub$Proxy@eb578a7
06-05 11:33:36.052 5599-5599/? I/HwLauncher: Launcher onclick end
06-05 11:33:36.171 15816-15816/myPackage W/linker: /data/app/myPackage-1/lib/arm/libutility.so has text relocations. This is wasting memory and prevents security hardening. Please fix.
06-05 11:33:36.193 15816-15816/myPackage I/[VCamera]: Application package name: myPackage
06-05 11:33:36.193 15816-15816/myPackage I/[VCamera]: Copyright (c) YIXIA (http://miaopai.yixia.com)
06-05 11:33:36.242 4212-4251/? I/Bluetooth_framework: BluetoothManagerService:Message: 20
06-05 11:33:36.242 4212-4251/? I/Bluetooth_framework: BluetoothManagerService:Added callback: android.bluetooth.IBluetoothManagerCallback$Stub$Proxy@c38275e:true pid = 15816
06-05 11:33:36.320 15816-15816/myPackage I/HwCust: Constructor found for class android.app.HwCustHwWallpaperManagerImpl
06-05 11:33:36.328 5554-6170/? W/PhoneInterfaceManager: shouldBlockLocation running ...
06-05 11:33:36.340 5554-6170/? I/PhoneInterfaceManager: shouldBlockLocation  ret:false
06-05 11:33:36.348 15816-15816/myPackage E/myPackage: >>>>>>>>>>>>>>>>>>>切到前台 activity process
06-05 11:33:36.353 5599-6072/? W/OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
06-05 11:33:36.353 5599-6072/? W/OpenGLRenderer: Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
06-05 11:33:36.395 15816-15872/myPackage I/System.out: [/system/xbin/which, su]
06-05 11:33:36.395 15816-15872/myPackage I/System.out: null
06-05 11:33:36.395 15816-15872/myPackage I/System.out: null
06-05 11:33:36.400 15816-15872/myPackage I/System.out: Calling by::className:com.umeng.commonsdk.internal.utils.e  MethodName:a
06-05 11:33:36.435 5538-6472/? W/PGApi_client: recv actoionId = 10013, action = com.huawei.pgmng.PGAction@ff71ced actionId =10013 pkg =com.android.mms extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.438 5538-6472/? W/PGMiddleWare: in handleAction method, action = 10013
06-05 11:33:36.438 5538-6472/? W/PGMiddleWare: in handleAction, invoke client = com.huawei.pgmng.middleware.AudioEffectLowPowerImpl@f7e3997, action = com.huawei.pgmng.PGAction@ff71ced actionId =10013 pkg =com.android.mms extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.444 5599-5599/? I/HwLauncher: Launcher onStop()
06-05 11:33:36.463 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10031
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: expired arrive. level:-1
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: during_ms:2006,period_ms:2000
06-05 11:33:36.560 4212-4212/? I/TrafficMonitor: count:983,rx_sum:2426,tx_sum:2401,rxBytes:2979752,during_ms:2006,rx_speed:1485419.7407776671,tx_speed:90334.99501495514,rto:1.0104123281965847
06-05 11:33:36.566 4602-4602/? I/NetworkSpeedManagerEx: mIsStop = false
06-05 11:33:36.567 15816-15877/myPackage E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:36.567 15816-15877/myPackage I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:36.571 4602-4602/? I/NetworkSpeedManagerEx:  value = 1122148.4
06-05 11:33:36.572 4602-4602/? I/NetworkSpeedManagerEx: speed = 1.1M/s
06-05 11:33:36.572 4602-4602/? I/NetworkSpeedView: /update(), speed=1.1M/s
06-05 11:33:36.614 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.615 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.615 4685-5353/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.627 20164-20164/? E/OsUtil: Buggle checkHasSmsPermissionsForUser Enable 10031
06-05 11:33:36.651 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.653 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.653 4685-15320/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.666 20164-20213/? E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:36.667 20164-20213/? I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:36.679 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.681 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.682 15816-15872/myPackage I/CameraManagerGlobal: Connecting to camera service
06-05 11:33:36.695 5538-6472/? W/PGApi_client: recv actoionId = 10000, action = com.huawei.pgmng.PGAction@355b722 actionId =10000 pkg =myPackage extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.696 5538-6472/? W/PGMiddleWare: in handleAction method, action = 10000
06-05 11:33:36.696 5538-6472/? W/PGMiddleWare: in handleAction, invoke client = com.huawei.pgmng.middleware.AudioEffectLowPowerImpl@f7e3997, action = com.huawei.pgmng.PGAction@355b722 actionId =10000 pkg =myPackage extend1 =20119 extend2 = flag =3 type =1
06-05 11:33:36.703 4685-5080/? I/HwSystemManager: NotificationGuideService:handle MSG_ACTIVIY_FOREGROUND, uid:10205
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.707 3809-6114/? I/FlipSensorManager: isFlipCameraPhone: ro.camera.flipsensor.enable not opened!
06-05 11:33:36.709 3809-3809/? I/CameraService: getCameraCharacteristics: Switching to HAL1 shim implementation...
06-05 11:33:36.710 3809-3942/? I/CameraService: getCameraCharacteristics: Switching to HAL1 shim implementation...
06-05 11:33:36.717 3664-3757/? I/[Gralloc]: alloc succ handle[0xb85692d8] stride[1120]
06-05 11:33:36.717 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.728 3664-5199/? I/[Gralloc]: alloc succ handle[0xb8568ed8] stride[1120]
06-05 11:33:36.728 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.740 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.742 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.743 4685-5396/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.763 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:URL= content://com.huawei.systemmanager.HarassmentInterceptionDBProvider/interception_messages
06-05 11:33:36.764 3664-3757/? I/[Gralloc]: alloc succ handle[0xb856ac78] stride[1120]
06-05 11:33:36.764 3664-3757/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.764 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:cursor count= 0
06-05 11:33:36.764 4685-5071/? I/HwSystemManager: HarassmentInterceptionDBProvider:queryUnReadCount: unread intercepted message count = 0
06-05 11:33:36.784 20164-20164/? E/Mms/cache: getSingleContact error for: 
06-05 11:33:36.785 20164-20164/? E/Mms: [1] Mms_conv
06-05 11:33:36.788 3664-5199/? I/[Gralloc]: alloc succ handle[0xb85bac20] stride[1120]
06-05 11:33:36.788 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.823 3664-3757/? I/[Gralloc]: alloc succ handle[0xb8606d88] stride[1120]
06-05 11:33:36.837 3664-5199/? I/[Gralloc]: alloc succ handle[0xb8594f40] stride[1120]
06-05 11:33:36.846 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.870 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84378f8] stride[1120]
06-05 11:33:36.870 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.891 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84dee18] stride[1120]
06-05 11:33:36.891 3664-5199/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:36.910 3664-5199/? I/[Gralloc]: alloc succ handle[0xb84ded38] stride[1120]
06-05 11:33:37.050 15816-15896/myPackage I/System.out: true
06-05 11:33:37.077 15816-15877/myPackage E/HAL: load: id=gralloc != hmi->id=gralloc
06-05 11:33:37.077 15816-15877/myPackage I/OpenGLRenderer: Initialized EGL, version 1.4
06-05 11:33:37.086 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.086 15816-15896/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.117 15816-15896/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.125 3664-3755/? I/[Gralloc]: alloc succ handle[0xb8602d30] stride[1120]
06-05 11:33:37.125 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.151 15816-15899/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.152 3664-3755/? I/[Gralloc]: alloc succ handle[0xb86150a8] stride[1120]
06-05 11:33:37.152 3664-3755/? I/[Gralloc]: alloc w[1080] h[1812] format[1] usage[2816]
06-05 11:33:37.174 3664-3755/? I/[Gralloc]: alloc succ handle[0xb8586db0] stride[1120]
06-05 11:33:37.184 15816-15899/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.233 15816-15899/myPackage W/System.err: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.240 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:328)
06-05 11:33:37.240 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.http.SocketConnector.connectTls(SocketConnector.java:103)
06-05 11:33:37.240 15816-15899/myPackage W/System.err:     at com.android.okhttp.Connection.connect(Connection.java:143)
06-05 11:33:37.241 15816-15899/myPackage W/System.err:     at com.android.okhttp.Connection.connectAndSetOwner(Connection.java:185)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.OkHttpClient$1.connectAndSetOwner(OkHttpClient.java:128)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.http.HttpEngine.nextConnection(HttpEngine.java:341)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:330)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:248)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:433)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect(HttpURLConnectionImpl.java:114)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getOutputStream(HttpURLConnectionImpl.java:245)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getOutputStream(DelegatingHttpsURLConnection.java:218)
06-05 11:33:37.242 15816-15899/myPackage W/System.err:     at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:25)
06-05 11:33:37.249 15816-15899/myPackage W/System.err:     at cn.jpush.android.util.s.a(Unknown Source)
06-05 11:33:37.249 15816-15899/myPackage W/System.err:     at cn.jpush.android.util.aj.b(Unknown Source)
06-05 11:33:37.250 15816-15899/myPackage W/System.err:     at cn.jpush.android.util.al.run(Unknown Source)
06-05 11:33:37.250 15816-15899/myPackage W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
06-05 11:33:37.250 15816-15899/myPackage W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
06-05 11:33:37.250 15816-15899/myPackage W/System.err:     at java.lang.Thread.run(Thread.java:833)
06-05 11:33:37.250 15816-15899/myPackage W/System.err: Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.250 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:324)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.TrustManagerImpl.checkServerTrusted(TrustManagerImpl.java:215)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at cn.jpush.android.util.u.checkServerTrusted(Unknown Source)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.Platform.checkServerTrusted(Platform.java:117)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.verifyCertificateChain(OpenSSLSocketImpl.java:556)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.NativeCrypto.SSL_do_handshake(Native Method)
06-05 11:33:37.251 15816-15899/myPackage W/System.err:     at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:324)
06-05 11:33:37.251 15816-15899/myPackage W/System.err: 	... 18 more
06-05 11:33:37.251 15816-15899/myPackage W/System.err: Caused by: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
06-05 11:33:37.253 15816-15899/myPackage W/System.err: 	... 25 more
06-05 11:33:37.263 4212-4252/? I/ActivityManager: Displayed myPackage/.t4.android.ThinksnsActivity: +578ms (total +1s790ms)
06-05 11:33:37.272 15816-15899/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.303 15816-15899/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.411 5538-5538/? I/K3V3CpuGovernorPolicy: close previously scene Hmp policy : 300 , 150
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: update:rxPkts:193,txPkts:220,rxBytes:176530,txBytes:17968
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: no expired. level:5
06-05 11:33:37.579 4212-4212/? I/TrafficMonitor: gettimer:interval=2000
06-05 11:33:37.580 4212-4212/? I/TrafficMonitor: during_ms:2002,period_ms:2000
06-05 11:33:37.580 4212-4212/? I/TrafficMonitor: count:984,rx_sum:193,tx_sum:220,rxBytes:176530,during_ms:2002,rx_speed:88176.82317682319,tx_speed:8975.024975024977,rto:0.8772727272727273
06-05 11:33:37.727 15816-15873/myPackage I/System.out: true
06-05 11:33:37.752 15816-15873/myPackage I/System: core_booster, getBoosterConfig = false
06-05 11:33:37.786 15816-15873/myPackage I/System: core_booster, getBoosterConfig = false

--- a/okdownload/src/main/java/com/liulishuo/okdownload/core/interceptor/connect/HeaderInterceptor.java
+++ b/okdownload/src/main/java/com/liulishuo/okdownload/core/interceptor/connect/HeaderInterceptor.java
@@ -86,14 +86,15 @@ public class HeaderInterceptor implements Interceptor.Connect {
 
         DownloadConnection.Connected connected = chain.processConnect();
 
+        if (chain.getCache().isInterrupt()) {
+            throw InterruptException.SIGNAL;
+        }
+
         Map<String, List<String>> responseHeaderFields = connected.getResponseHeaderFields();
         if (responseHeaderFields == null) responseHeaderFields = new HashMap<>();
 
         OkDownload.with().callbackDispatcher().dispatch().connectEnd(task, blockIndex,
                 connected.getResponseCode(), responseHeaderFields);
-        if (chain.getCache().isInterrupt()) {
-            throw InterruptException.SIGNAL;
-        }
 
         // if precondition failed.
         final DownloadStrategy strategy = OkDownload.with().downloadStrategy();

